network-store-server/src/test/java/com/powsybl/network/store/server/NetworkStoreControllerIT.java

```diff
@@ -849,19 +845,6 @@ void test() throws Exception {
                 .andExpect(jsonPath("data[0].attributes.q").value(40))
                 .andExpect(jsonPath("data[0].attributes.node").value(6));
 
-        // Test removals
-        mvc.perform(delete("/" + VERSION + "/networks/" + NETWORK_UUID + "/" + Resource.INITIAL_VARIANT_NUM + "/switches/b1")
-                .contentType(APPLICATION_JSON))
-                .andExpect(status().isOk());
-
-        mvc.perform(delete("/" + VERSION + "/networks/" + NETWORK_UUID + "/" + Resource.INITIAL_VARIANT_NUM + "/voltage-levels/baz")
-                .contentType(APPLICATION_JSON))
-                .andExpect(status().isOk());
-
-        mvc.perform(delete("/" + VERSION + "/networks/" + NETWORK_UUID + "/" + Resource.INITIAL_VARIANT_NUM + "/substations/bar")
-                .contentType(APPLICATION_JSON))
-                .andExpect(status().isOk());
-
```

Comment:
Can we also keep the delete for this one?

Reply:
done